sim/field.ts

```diff
@@ -103,11 +103,16 @@ export class Field {
 	}
 
 	suppressingWeather() {
+		const strongWeathers = ['desolateland', 'primordialsea', 'deltastream'];
```

Comment:
Changes in sim/field.ts should no longer be necessary with us ending the weather properly.